packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -87,11 +87,13 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
     const defaultStrings = ColorPickerBase.defaultProps.strings as Required<IColorPickerStrings>;
 
     this._textLabels = {
-      r: props.redLabel || strings.red || defaultStrings.red,
-      g: props.greenLabel || strings.green || defaultStrings.green,
-      b: props.blueLabel || strings.blue || defaultStrings.blue,
-      a: props.alphaLabel || strings.alpha || defaultStrings.alpha,
-      hex: props.hexLabel || strings.hex || defaultStrings.hex
+      // tslint:disable:deprecation
+      r: strings.red || props.redLabel || defaultStrings.red,
```

Comment:
This will cause a problem because of the default props thing I mentioned in another comment. Same for the other color stuff.

Reply:
I just checked and it seems that this is the only file where that was happening. I've changed it so that it reverts to its previous implementation 👍